mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -271,41 +382,34 @@ export class CompareRunView extends Component {
     });
 
     return keys.map((k) => {
-      let row_class = undefined;
-      if (highlightChanges) {
-        const all_equal = data[k].every((x) => x === data[k][0]);
-        if (!all_equal) {
-          row_class = 'row-changed';
-        }
+      const all_equal = data[k].every((x) => x === data[k][0]);
```

Comment:
```suggestion
      const allEqual = data[k].every((x) => x === data[k][0]);
```

Can we use camel case for variable names?